projects/README.md

```diff
@@ -0,0 +1,101 @@
+# Group project
+
+## Objectives
+
+The objective of project work is to apply the knowledge gained in the course, in a group setting, to a selected (open) information retrieval problem.
```

Comment:
"project work is to apply" => "project work was to apply"

Reply:
Done